components/user-nav.tsx

```diff
@@ -100,6 +103,10 @@ export default function UserNav({ user }: any) {
               <span>Support</span>
             </DropdownMenuItem>
           </Link>
+          <DropdownMenuItem className="flex items-center justify-between cursor-pointer" onClick={() => setTheme(theme === 'light' ? 'dark' : 'light')} aria-label="Toggle theme">
```

Comment:
Just use `cursor-pointer` here, the other class names are making the spacing weird.